src/sentry/incidents/logic.py

```diff
@@ -1382,6 +1389,27 @@ def translate_aggregate_field(aggregate, reverse=False):
     return aggregate
 
 
+def trigger_sentry_app_action_creators_for_incidents(validated_alert_rule_data):
+    from rest_framework import serializers
```

Comment:
We have to do import shenanigans here because this file is absurdly large. One thing you can do here is move these two functions to a new file: `src/sentry/incidents/utils/sentry_apps.py`.

Reply:
Refactored!